doc/level1.rst

```diff
@@ -6,22 +6,42 @@
 Autocrypt Level 1: Enabling encryption, avoiding annoyances
 ===========================================================
 
-Autocrypt makes it easy for people to encrypt email.  This document
-describes the basic capabilities required for a mail app to be
-Autocrypt-capable at Level 1.
-
-The design of Level 1 is driven by usability concerns and by the
-realities of incremental deployment. A user may mix both
-Autocrypt-enabled MUAs and traditional MUAs, and we'd
-like to avoid annoyances like unexpectedly unreadable mails while also
-supporting users who want to explicitly turn on encryption.
-
-For ease of implementation and deployment, Level 1 focuses on the use
-of Autocrypt on a single device.  We intend to :doc:`support
-multi-device synchronization (and other features) as part of Level
-2<next-steps>`.  We want to keep Level 1 simple enough that it's easy
-for developers to adopt it so we can drive efforts from real-life
-experiences as soon as possible.
+Autocrypt aims to incrementally replace cleartext e-mail with end-to-end
+encrypted e-mail. The Level 1 specification makes it easy for users to
+encrypt. There are no changes required from e-mail providers and there
+are no dependencies on key servers. The spec describes the basic
+capabilities required for a mail app to be Autocrypt-capable at Level 1,
+allowing it to exchange end-to-end encrypted e-mails with other
+Autocrypt-capable mail apps.
+
+This specification is the result of maybe 2000 hours of talking and
+coding iterations during 2017. Around 20 people and many more users and
+trainers participated in Autocrypt sessions. The majority of these
+participants have engaged with the e-mail encryption space for a
+long time. There was overwhelming agreement that fresh approaches focusing
+on usability concerns are needed.
+
+Autocrypt aims to sustainably grow the overall percentage of end-to-end
+encrypted messages relayed via the e-email provider network. This
+differs from the focus on maximizing the security of an individual mail
+communication. Another major difference in approach is that Autocrypt Level 1
+only defends against passive data collection. Protection against active
+adversaries (those which modify messages in transit) is the aim of
+future specifications. We share and support :rfc:`the new perspective
+stated in RFC7435 ("Opportunistic Security: Some Protection Most of the
+Time") <7435#section-1.2>`.
+
+For ease of implementation and deployment, Level 1 focuses on the use of
+Autocrypt on a single device. Users get basic support for transfering
+secret keys between Autocrypt-capable mail apps. They need to enter a
+long number for securing the transfer, technically realized through
+sending and receiving an e-mail message. We aim to improve the usability
+around secret key sharing in subsequent Autocrypt specifications.
```

Comment:
I hope that future autocrypt specifications will aim to improve the multi-device use case -- not just secret key sharing.  we should stay focused on the user concerns; key management is a means to an end.